### DISCUSSION TITLE
Prevent table from re-sorting on data updates

### DISCUSSION THREAD

**Discussion by tukwan** [TanStack/table#5217] [Q&A]
How to stop sorting from being reapplied after the table data updates (input cell fields)? 
If I already have a column sorted and then reset sort, the rows will become unsorted (and still shift positions). Is there a clean way to stop sort on updates, or do I need to implement manual sorting?

- [Similar in v7
](https://stackoverflow.com/questions/62450548/how-to-prevent-react-table-sort-and-search-reset)
- https://github.com/TanStack/table/discussions/4592

**Comment by teelur** [comment]
Has anyone found an answer to this? I'm not sure why a feature like this would just be removed in a version upgrade...

**Comment by sammyhajomar** [comment]
Did you find a solution for this? @tukwan @tshea113

**Comment by SpringsTea** [comment]
Also looking for a solution here, other than manageing filtering and sorting myself.
